src/doc_builder/convert_md_to_mdx.py

```diff
@@ -25,11 +25,11 @@ def convert_md_to_mdx(md_text, page_info):
     """
     return """<script>
 import Tip from "./Tip.svelte";
-import Youtube from "./Youtube.svelte";	
-import Docstring from "./Docstring.svelte";	
-import CodeBlock from "./CodeBlock.svelte";	
-import CodeBlockFw from "./CodeBlockFw.svelte";	
-import IconCopyLink from "./IconCopyLink.svelte";	
```

Comment:
Are those trailing spaces absolutely necessary @mishig25 ? There is none on the first line, so I removed them since the quality script does not like trailing spaces.